src/main/java/uk/gov/hmcts/cmc/claimstore/events/ClaimIssuedEvent.java

```diff
@@ -47,15 +54,17 @@ public boolean equals(final Object other) {
         }
 
         final ClaimIssuedEvent that = (ClaimIssuedEvent) other;
+
         return Objects.equals(submitterEmail, that.submitterEmail)
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Reduce the number of conditional operators (4) used in the expression (maximum allowed 3). [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AS1067)